ProcessLib/RichardsMechanics/RichardsMechanicsFEM-impl.h

```diff
@@ -824,6 +878,67 @@ void RichardsMechanicsLocalAssembler<ShapeFunctionDisplacement,
         .noalias() += Kup * p_L;
 }
 
+template <typename ShapeFunctionDisplacement, typename ShapeFunctionPressure,
+          typename IntegrationMethod, int DisplacementDim>
+std::size_t RichardsMechanicsLocalAssembler<
+    ShapeFunctionDisplacement, ShapeFunctionPressure, IntegrationMethod,
+    DisplacementDim>::setKelvinVector(double const* values,
+                                      KelvinVectorType IpData::*member)
+{
+    auto const kelvin_vector_size =
+        MathLib::KelvinVector::KelvinVectorDimensions<DisplacementDim>::value;
+    auto const n_integration_points = _ip_data.size();
+
+    auto matrix_mapped_values =
```

Comment:
I wonder if `matrix_mapped_values` could be `const`? What happens in 
```C++
MathLib::KelvinVector::symmetricTensorToKelvinVector(
                matrix_mapped_values.col(ip));
```
?

Reply:
const added. Thanks.